content/storage-history/history.textile

```diff
@@ -0,0 +1,585 @@
+---
+title: History
+meta_description: "Learn about accessing message history with the history and rewind features"
+languages:
+ - javascript 
+ - java 
+ - python 
+ - php 
+ - ruby 
+ - nodejs 
+ - objc
+ - swift
+ - go 
+ - csharp
+---
+
+History is available for both realtime and REST interfaces, enabling you to retrieve previously sent messages and presence event history for channels. By default two minutes of message history is available to retrieve, but this can be extended by configuring the "storage options":/message-storage/storage for an application. It is also possible to persist messages to external storage services using "integrations":/integrations.
+
+h2(#history-rewind). History vs rewind
+
+You can retrieve previously published messages using the history feature or using the "rewind channel option":/channels/options/rewind. There are several differences between the two features that are important to be aware of when deciding how to retrieve older messages:
+
+* The number of messages that are returned varies between history and rewind. History returns up to 1000 messages. Rewind returns up to 100 messages.
+* History can be called using either the realtime or REST interface of an SDK. Rewind can only be used with the realtime interface of an SDK.
+* You can define a custom start or end time period for history. Rewind can return a specified number of messages, or a specified number of seconds/minutes, that immediately precede the moment of attachment.
```

Comment:
Should this one follow the point about message limits and repeated calls to history?

```suggestion
* You can define a custom start and end time to retrieve messages from using history. Rewind returns either a given number of messages, or messages up to a point in time in the past.
```